assignment6/Application.java

```diff
@@ -0,0 +1,231 @@
+package assignment6;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Scanner;
+
+public class Application {
+	public static void main(String[] args) {
+		Travel travel = new Travel();
+		try (Scanner inp = new Scanner(System.in)) {
+			System.out.println("Enter the speed of the Vehicle in km/hour : ");
+			travel.setSpeed( inp.nextInt());
+			
+			System.out.println("Enter the distance to travel by the Vehicle in km : ");
+			travel.setDistance( inp.nextInt());
+		}
+		DestinationTime dt = new DestinationTime();
+		dt.destination(travel);
+	}
+}
+
+
+class DestinationTime{
+	
+	CurrDateTime currDateTime = new CurrDateTime();
+	dayOfTheDate day_of_the_date= new dayOfTheDate();
+	int day=day_of_the_date.dayofweek( currDateTime.getDate(), currDateTime.getMonth(), currDateTime.getYear());
+	int desDate=currDateTime.getDate(),desMonth=currDateTime.getMonth(),desYear=currDateTime.getYear();
+	
+	int desHour=0;
+	float desMin=0;
+	public boolean isLeapYear(int year){
+		if(year%400==0)
+			return false;
+		else if (year%100==0)
+			return false;
+		else if (year%4==0) {
+			return true;
+		}
+		return false;	
+	}
+	public void destination(Travel travel)
+	{
+		float speed=travel.getSpeed();
+		float distance=travel.getDistance();
+		float RemTimeToTravel=distance/speed;
```

Comment:
Variables names should follow camelCasing as per Java standards. 
`RemTimeToTravel`, and `day_of_the_date` needs to be updated